.travis.yml

```diff
@@ -12,28 +12,17 @@ jobs:
     - php: 7.1
     - php: 7.2
     - php: 7.3
```

Comment:
Shouldnt we add 7.4 here now instead of nightly?

Reply:
Not available on Travis, that's why I added workflow for GitHub actions.

Reply:
Is it time to wait before we do this or?

Reply:
I think we should merge GitHub actions first

Reply:
Done

Reply:
👍